### ISSUE TITLE
Cannot upload on GoCD 20.1.0

### ISSUE THREAD

**Issue by paambaati** [indix/gocd-s3-artifacts#74]
I'm trying out the plugin on GoCD 20.1.0, and I see that publish to S3 is failing.

```
2020-02-03 07:50:40,618 ERROR [scheduler-3] Builder:114 - javax/xml/bind/JAXBException
java.lang.RuntimeException: javax/xml/bind/JAXBException
	at com.thoughtworks.go.plugin.infra.FelixGoPluginOSGiFramework.executeActionOnTheService(FelixGoPluginOSGiFramework.java:214)
	at com.thoughtworks.go.plugin.infra.FelixGoPluginOSGiFramework.doOn(FelixGoPluginOSGiFramework.java:164)
	at com.thoughtworks.go.plugin.infra.DefaultPluginManager.submitTo(DefaultPluginManager.java:131)
	at com.thoughtworks.go.plugin.access.PluginRequestHelper.submitRequest(PluginRequestHelper.java:49)
	at com.thoughtworks.go.plugin.access.pluggabletask.JsonBasedTaskExecutor.execute(JsonBasedTaskExecutor.java:41)
	at com.thoughtworks.go.domain.builder.pluggableTask.PluggableTaskBuilder.executeTask(PluggableTaskBuilder.java:88)
	at com.thoughtworks.go.domain.builder.pluggableTask.PluggableTaskBuilder.lambda$build$0(PluggableTaskBuilder.java:67)
	at com.thoughtworks.go.plugin.access.pluggabletask.TaskExtension.execute(TaskExtension.java:58)
	at com.thoughtworks.go.domain.builder.pluggableTask.PluggableTaskBuilder.build(PluggableTaskBuilder.java:67)
	at com.thoughtworks.go.remote.work.Builders.build(Builders.java:73)
	at com.thoughtworks.go.remote.work.BuildWork.execute(BuildWork.java:208)
	at com.thoughtworks.go.remote.work.BuildWork.buildJob(BuildWork.java:181)
	at com.thoughtworks.go.remote.work.BuildWork.build(BuildWork.java:129)
	at com.thoughtworks.go.remote.work.BuildWork.doWork(BuildWork.java:79)
	at com.thoughtworks.go.agent.JobRunner.run(JobRunner.java:53)
	at com.thoughtworks.go.agent.AgentHTTPClientController.retrieveWork(AgentHTTPClientController.java:147)
	at com.thoughtworks.go.agent.AgentHTTPClientController.work(AgentHTTPClientController.java:119)
	at com.thoughtworks.go.agent.AgentController.loop(AgentController.java:84)
	at jdk.internal.reflect.GeneratedMethodAccessor9.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.run(ScheduledMethodRunnable.java:65)
	at org.springframework.scheduling.support.DelegatingErrorHandlingRunnable.run(DelegatingErrorHandlingRunnable.java:54)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.runAndReset(Unknown Source)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: java.lang.NoClassDefFoundError: javax/xml/bind/JAXBException
	at com.amazonaws.util.Md5Utils.md5AsBase64(Md5Utils.java:104)
	at com.amazonaws.services.s3.AmazonS3Client.putObject(AmazonS3Client.java:1598)
	at com.indix.gocd.utils.store.S3ArtifactStore.put(S3ArtifactStore.java:74)
	at com.indix.gocd.utils.store.S3ArtifactStore.put(S3ArtifactStore.java:64)
	at com.indix.gocd.s3publish.PublishExecutor$1.execute(PublishExecutor.java:110)
	at com.indix.gocd.s3publish.PublishExecutor$1.execute(PublishExecutor.java:104)
	at com.indix.gocd.utils.utils.Functions$VoidFunction.apply(Functions.java:8)
	at com.indix.gocd.utils.utils.Functions$VoidFunction.apply(Functions.java:4)
	at com.indix.gocd.utils.utils.Lists.foreach(Lists.java:12)
	at com.indix.gocd.s3publish.PublishExecutor.pushToS3(PublishExecutor.java:104)
	at com.indix.gocd.s3publish.PublishExecutor.execute(PublishExecutor.java:54)
	at com.indix.gocd.s3publish.PublishTask.handleTaskExecution(PublishTask.java:75)
	at com.indix.gocd.s3publish.PublishTask.handle(PublishTask.java:47)
	at com.thoughtworks.go.plugin.infra.DefaultPluginManager.lambda$submitTo$0(DefaultPluginManager.java:134)
	at com.thoughtworks.go.plugin.infra.FelixGoPluginOSGiFramework.executeActionOnTheService(FelixGoPluginOSGiFramework.java:208)
	... 28 common frames omitted
Caused by: java.lang.ClassNotFoundException: javax.xml.bind.JAXBException not found by indix.s3publish [4]
	at org.apache.felix.framework.BundleWiringImpl.findClassOrResourceByDelegation(BundleWiringImpl.java:1639)
	at org.apache.felix.framework.BundleWiringImpl.access$200(BundleWiringImpl.java:80)
	at org.apache.felix.framework.BundleWiringImpl$BundleClassLoader.loadClass(BundleWiringImpl.java:2053)
	at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
	... 43 common frames omitted
```

If there's additional information I can provide, let me know.

**Comment by paambaati** [comment]
If https://github.com/indix/gocd-s3-artifacts/pull/71 fixes this issue, would you folks mind cutting a new release?

CC: @manojlds @ashwanthkumar @brewkode

**Comment by manojlds** [comment]
Published v0.6.0 release

**Comment by paambaati** [comment]
@manojlds Did you mean to publish `v6.0.0`? Because the last published version was `v5.2.1`.

**Comment by manojlds** [comment]
Yeah have fixed it to be v6.0.0
